docs/spec/README.md

```diff
@@ -14,21 +14,21 @@ please submit them to our [bug bounty](https://tendermint.com/security)!
 
 ### Data Structures
 
-- [Encoding and Digests](./blockchain/encoding.md)
-- [Blockchain](./blockchain/blockchain.md)
-- [State](./blockchain/state.md)
+- [Encoding and Digests](https://github.com/tendermint/tendermint/blob/master/docs/spec/blockchain/encoding.md)
```

Comment:
oh goodness! I'm sorry, by absolute I meant with respect to the repo. so just `/docs/spec/...` is fine. as long as the first `/` means `top of this repo`. Full links is probably excessive